### DISCUSSION TITLE
delightful model training, quality question

### DISCUSSION THREAD

**Discussion by pivolan** [coqui-ai/TTS#2817] [General Q&A]
I have already trained delightful model on 2.5 hours dataset from the scratch, and results are worst.
Also I tried to do the same on vits model, and results are much better. 
Can I train separately vocoder for delightful? And with what config vits trained for vocoder? 

This is my examples of audio:
Text: Das Imperium Spartherm. Tiefer Einblick in die Meller Firma liefert neue Erkenntnisse. Gegen den Meller Kaminofenhersteller Spartherm wird wegen Betrugs und Untreue ermittelt. Hinter dem Namen steckt ein kompliziertes Imperium mit allerhand Verstrickungen. Wir bringen Licht ins Dunkel.

delightful 1000 epochs:
https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/de_delight_1000epochs_2.5hours.wav
vits: https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/vits_1000epochs_2.5h.wav
vits thorsten my train: https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/thorsten_vits_1000epochs_22hours.wav
vits thorsten, exists pretrained model in coqui repo: https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/vits_th_inner_model.wav

I cannot understand why is so many noise in delightful model?

**Comment by pivolan** [comment]
my config for delightful:

```
    run_name=RUN_NAME,
    run_description="Train delightful tts.",
    model_args=model_args,
    audio=audio_config,
    vocoder=vocoder_config,
    batch_size=12,
    eval_batch_size=12,
    num_loader_workers=10,
    num_eval_loader_workers=10,
    precompute_num_workers=10,
    batch_group_size=2,
    compute_input_seq_cache=True,
    compute_f0=True,
    f0_cache_path=os.path.join(output_path, RUN_NAME + "f0_cache"),
    run_eval=False,
    test_delay_epochs=-1,
    epochs=1000,
    text_cleaner="phoneme_cleaners",
    use_phonemes=True,
    phoneme_language="de",
    phonemizer='espeak',
    phoneme_cache_path=os.path.join(output_path, RUN_NAME + "phoneme_cache"),
    print_step=50,
    log_model_step=1000,
    save_step=100,
    save_n_checkpoints=4,
    save_checkpoints=True,
    print_eval=False,
    mixed_precision=True,
    output_path=output_path,
    datasets=[dataset_config],
    start_by_longest=True,
    eval_split_size=0.1,
    binary_align_loss_alpha=0.0,
    use_attn_priors=False,
    max_text_len=110,
    #steps_to_start_discriminator=10000,
    test_sentences=[ "Ich gehe gerne spazieren, besonders im Frühling.",

```

**Comment by pivolan** [comment]
May be there are other receipts to train vocoder separately? Now it is the same effect that I can hear when use fastpitch + hifigan, when vocoder trained not enough.

**Comment by erogol** [comment]
you need to fine-tune the vocoder. There is no compatible one currently.

**Comment by thorstenMueller** [comment]
Hi @pivolan , your VITS model from your first post is sounding really impressive (like human read) 👏
